core/services/chainlink/secret_generator_test.go

```diff
@@ -0,0 +1,28 @@
+package chainlink
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSecretGenerator_sessionSecret(t *testing.T) {
```

Comment:
I think your test `FilePersistedSecretGenerator` specifically, the test name should be `TestFilePersistedSecretGenerator` probably.